infrastructure/prod.tfvars

```diff
@@ -7,3 +7,4 @@ idam_web_service = "https://hmcts-access.service.gov.uk"
 rd_professional_api_service = "http://rd-professional-api-prod.service.core-compute-prod.internal"
 s2s_service = "http://rpe-service-auth-provider-prod.service.core-compute-prod.internal"
 fee_and_pay_api = "http://payment-api-prod.service.core-compute-prod.internal"
+iss_service = "https://forgerock-am.service.core-compute-idam-prod.internal:8443/openam/oauth2/hmcts"
```

Comment:
as before, need to make sure what prod iss value is

Reply:
@paddybasi we are not going live with this feature now. I still need another refactor in the next sprint. But, we still need to find out whats the iss value